specifications/transactions/travel-rule/README.md

```diff
@@ -0,0 +1,372 @@
+# Off-CHain Travel Rule Exchange
+
+This protocol is used to exchange KYC information about a potential transaction between to parties that will be resolved on-chain upon its completion. This is a dual attestation, preflight protocol. At the end of the protocol, the sender will receive a signature from the recipient so that the Diem Paymnet Network can verify that the tranasfer has been signed off by both the sender and recipient. Some considerations:
+* Participants that do not implement this cannot tranfser amounts above the travel rule limit
```

Comment:
```suggestion
* Participants that do not implement this cannot transfer amounts above the travel rule limit
```

Reply:
This seems a bit harsh no?  Isn't it rather than this is a potential protocol for transferring amounts above the travel rule limit?

Reply:
@aching If we aren't going to make this *the* standard for transacting on diem then what's the point? Just let every VASP figure it out themselves and we'll go and work on other things.

Reply:
above => equal or above

Reply:
I agree we want people to opt-in to this protocol (as no other option currently exists).  However, it is not required for DPN members specifically AFAIK.  The summary of DIP-1 indicates this to me as well - https://github.com/diem/dip/blob/master/dips/dip-1.mdx#summary

Reply:
I think there are three things:
* This is the only standard supported
* One must participate in some dual attestation protocol
* We need another doc that actually covers transaction flows, since this is not a transaction flow